defaults/main.yml

```diff
@@ -15,4 +11,4 @@ dokku_vhost_enable: 'true'
 dokku_web_config: 'false'
 herokuish_version: 0.5.5
 plugn_version: 0.3.2
-sshcommand_version: 0.9.0
+sshcommand_version: 0.8.0
```

Comment:
Yeah looks like we didn't update `defaults/main.yml.base`. Mind fixing there and regenerating?

Reply:
Can you update the valie in `defaults/main.yml.base` to be `0.9.0` and regenerate? Sorry if that was not clear.